packages/gafl-webapp-service/src/routes/misc-routes.js

```diff
@@ -92,7 +92,7 @@ export default [
     path: ADD_PERMISSION.uri,
     handler: async (request, h) => {
       await addPermission(request)
-      return h.redirect(CONTROLLER.uri)
```

Comment:
Need tests to cover these changes, please (existing tests obviously just run the code without asserting anything)